Monika After Story/game/script-ch30.rpy

```diff
@@ -1361,6 +1370,9 @@ label ch30_preloop:
     $ persistent._mas_game_crashed = True
     $ startup_check = False
     $ mas_checked_update = False
+    $ mas_globals.last_minute_dt = datetime.datetime.now()
```

Comment:
Can we make this a persistent var so the daily check will properly run daily?

For what I need this to do, I need the daily check to run once a day, but with this setup, if I load in today, then say goodbye before midnight, then load tomorrow, the daily check does not run, meaning I need to duplicate code in the ch30_reset too.

This would also be beneficial for those who have average session times under 1 hour, as they would be sure to run

Reply:
That's not what this is intended for, though. This is meant to act the same as the once per minute checks, just with different time intervals. If you need something to run in either ch30 reset or the day check, then just define the check as a label/function elsewhere and call it in both places.

Reply:
Still though, why not be able to handle both cases?

Reply:
because thats not what the minute check does. It doesn't run immediately on start, it only runs after the first minute. any checks from that case that we want to run initially (like the Event conditionals check) is run in preloop or before preloop.